model-optimizer/unit_tests/mo/utils/cli_parser_test.py

```diff
@@ -967,3 +967,216 @@ def test_check_low_latency_is_available(self, available_transformations):
     def test_check_dummy_pass_is_available(self, available_transformations):
         available_transformations.return_value = {"LowLatency2": None}
         self.assertRaises(Error, check_available_transforms, [("DummyPass", "")])
+
+
```

Comment:
Will we check layout setting in e2e tests? Could you cover add_layouts function in unit tests?

Reply:
Yes, it will be tested by e2e tests